docs/tutorials/protect_against_regen.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _protect_against_regen:
+
+*************************************************************************
+Configuring Your Module to Protect Against Regeneration Voltage Spikes
+*************************************************************************
+
+What are Regenerative Voltage Spikes
+=======================================
+Regeneration voltage occurs when the commanded module voltage is less than the module's back-EMF voltage. When this occurs, the motor's kinetic 
+energy is converted directly to electrical energy, and produces a current flowing out of the motor and into the power source.
+
+How Can Regeneration Affect Your Module
+=========================================
+If the module's power source can absorb the regenerated current without allowing the voltage to spike, regeneration protection isn't needed. This is often true with a battery as the power source.
+When connected to a less than constant voltage source, like a benchtop power supply, the overall system voltage will increase during regeneration. 
+These spikes can result in voltages above the module's and your supply's maximum rated operating voltage. When this happens, you can permanently damage the module, the power supply, or both. 
+When this happens, you can permanently damage the module, the power supply, or both.
+
+Protecting Against Dangerous Regeneration Voltage Spikes
+===========================================================
+Regenerative voltage spikes pose a serious threat to both your modules and your power supply. As such, all Vertiq modules are able to self-limit the amount of 
+regeneration voltage produced. Vertiq modules have two main parameters to protect against dangerous regeneration voltage spikes. They are available through IQ Control Center's advanced tab as 
+*Volts Limit* and *Volts Limit Starting Voltage*.
+
+For example, by default on Vertiq 81-XX family modules (rated to 14S), you will see:
+
+.. image:: ../_static/control_center_pics/regen_params.png
+        :alt: Regeneration Parameters in IQ Control Center
+
+In short, these two values work together in order to limit the rate at which your module will slow down or apply a braking force. In doing so, the module limits the amount of 
+negative supply current produced, reducing the produced regenerative voltage.
```

Comment:
In the same vein as my comment above about clarifying the power supplies role in this, I may just tweak this last bit to be "...reducing the voltage spike produced by the power supply